README.md

```diff
@@ -19,3 +19,35 @@ This code can be profiled for coverage. See [COVERAGE.md].
 
 [COVERAGE.md]:    ./COVERAGE.md
 
+squeezed: a xen host memory ballooning daemon
+---------------------------------------------
+
+Squeezed uses [ballooning](http://static.usenix.org/events/osdi02/tech/full_papers/waldspurger/waldspurger_html/node6.html)
+to move memory between running VMs. It is able to:
+
+  1. avoid wasting host memory: unused memory can be gifted to VMs
+  2. share memory according to a configured policy, so some VMs will use more than others
+  3. "squeeze" existing VMs to make room to start new VMs.
+
+Squeezed is an optional component of the [xapi toolstack](http://wiki.xen.org/wiki/Choice_of_Toolstacks).
+
+building the development version
```

Comment:
Let's delete these instructions, which aren't right anymore.

Reply:
These incorrect instructions are still there.

Reply:
documentation about coverage is also out of date